tests/integration/models_images_test.py

```diff
@@ -87,8 +87,14 @@ def test_pull_with_sha(self):
     def test_pull_multiple(self):
         client = docker.from_env(version=TEST_API_VERSION)
         images = client.images.pull('hello-world')
-        assert len(images) == 1
-        assert 'hello-world:latest' in images[0].attrs['RepoTags']
+        assert len(images) >= 1
+        found = False
+        for img in images:
```

Comment:
Could you use `any()` instead of doing a for loop. E.g. something like `assert any('hello-world:latest' in img[0].attrs['RepoTags'] for img in images)`?

Reply:
Ah, yes, sure. Didn't know of that approach 🤗

Reply:
It's just `img.attrs`, not `img[0].attrs`.